tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -167,6 +167,26 @@ var _ = Describe("Create machinepool",
 
 			})
 
+		It("validate inputs for create spot machinepool - [id:43252]",
+			labels.Medium,
+			labels.Day1Negative,
```

Comment:
machine pool is day2

Reply:
@radtriste I added a `day2-negative` label for it now

Reply:
you did not need to :) `Day2` label was enough. We don't run specific Day2 negative job, they are run as part of Day2

Reply:
Makes sense. Updated the label again. 

Also, do we have some docs somewhere that explain what qualifies as a day1, day2, etc test? I seem to be misjudging what tests qualify for each label